css/src/components/table.scss

```diff
@@ -64,4 +67,59 @@ $table-caption-spacing: $letter-spacing-medium !default;
 			}
 		}
 	}
+
+	&.table-simple {
+		border: none;
+	}
+
+	&.table-borderless {
+		border: none;
+
+		th,
+		td {
+			border: none;
+		}
+	}
+
+	&.table-complex {
+		thead tr,
+		tfoot tr {
+			background-color: $table-header;
+		}
+
+		th,
+		td {
+			border: 1px solid $table-border-dark;
+		}
+	}
+
+	&.table-striped {
+		tbody tr:nth-of-type(even) {
+			background-color: $table-row;
+		}
+	}
+
+	&.table-sm {
+		th,
+		td {
+			padding: $table-cell-padding-compact;
+		}
+	}
+
+	&.table-lg {
+		th,
+		td {
+			padding: $table-cell-padding-loose;
+		}
+	}
+
+	&.table-width-reset {
+		width: auto;
+	}
+
+	&.table-interactive {
```

Comment:
I think we should remove `tablet-interactive` for now as well. We don't have direct use case for this right now, and don't know if it may require JS eventually.

Reply:
In the [tables discussion](https://github.com/microsoft/atlas-design/discussions/170#discussioncomment-847305) Jeremy mentioned there is a need in highlighting the row, so I though we can do at leat the hover effect. And later figure out what to do (or not do) with keyboard focusing on the rows.

Reply:
Row highlighting is quite different from a hover effect background color change. Still seems best to me that we remove this until a feature really requires it and we have a better understanding of it's accessibility concerns.

Reply:
Sounds good, will remove it!

Reply:
Updated